ElectronicShop/Models/Interfaces/IPasswordService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Security.Cryptography.X509Certificates;
+
+namespace ElectronicShop.Models.Interfaces
+{
+    public interface IPasswordService
+    {
+        string EncryptString();
+        string DecryptString(string encpasw);
```

Comment:
Reikėtų netrumpinti žodžių arba trupinti kai tai visiems suprantama, bet geriau išvis netrumpinti. Aš dabar nesuprantu kas paduodama į metodą.

Reply:
Čia paduodamas užšifuotas salaptažodis, o to kai darys log in tikrins ar atitinka

Reply:
pakeista

Reply:
interface pakeista, o klasėje nepakeista